api/auth.py

```diff
@@ -20,12 +20,22 @@ class TokenData(BaseModel):
     username: Optional[str] = None
 
 
+class Settings(BaseSettings):
+    secret_key: str
+
+    class Config:
+        env_file = "../.env"
+
+
 class Authentication:
     # ToDo: use settings https://fastapi.tiangolo.com/advanced/settings/
 
+    settings = Settings()
```

Comment:
Please move this to the `Authentication()` constructor:
```python
    def __init__(self, db):
        self._pwd_context = CryptContext(schemes=["bcrypt"], deprecated="auto")
        self._db = db
        self._settings = Settings()
```
Then `SECRET_KEY` can be replaced with `self._settings.secret_key` in the class methods.

As the settings are loaded dynamically, they should be a member of each `Authentication` instance which is created at runtime rather than a static member of the class which is created only once when the Python module is loaded.